<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>db_show</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>520</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
	
	color: rgb(255, 255, 255);
	background-color: rgb(56, 56, 56);
}
::section {
	background-color: rgb(56, 56, 56);
	color: rgb(255, 255, 255);

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0.125, x2:1, y2:1, stop:0 rgba(12, 26, 31, 255), stop:1 rgba(88, 87, 121, 255));
border-color: qlineargradient(spread:pad, x1:0, y1:0.125, x2:1, y2:1, stop:0 rgba(12, 26, 31, 255), stop:1 rgba(88, 87, 121, 255));</string>
   </property>
   <property name="text">
    <string>Хотите увидеть свой результат? 
Введите свои имя и фамилию</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_surname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>225</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(100, 98, 137);
border-bottom-right-radius: 15px;
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="placeholderText">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(100, 98, 137);
border-top-right-radius: 15px;
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>40</number>
   </property>
   <property name="placeholderText">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0.125, x2:1, y2:1, stop:0 rgba(12, 26, 31, 255), stop:1 rgba(88, 87, 121, 255));
	color: rgb(255, 255, 255);
	border: 2px solid black;
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.034, x2:1, y2:1, stop:0 rgba(0, 3, 1, 255), stop:1 rgba(31, 69, 47, 255));
	border: 4px solid black;
	color: rgb(51, 182, 31);

}</string>
   </property>
   <property name="text">
    <string>Посмотреть результаты</string>
   </property>
  </widget>
  <widget class="QLabel" name="fon_lb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_bt">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>5</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
	background-color: rgb(202, 202, 202);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	
	background-color: rgb(143, 143, 143);
	color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/back.png</normaloff>../images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_bt">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(65, 65, 65);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../images/user.png</normaloff>../images/user.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="sh_lb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(65, 65, 65);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>       User's result</string>
   </property>
  </widget>
  <zorder>fon_lb</zorder>
  <zorder>table</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_surname</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>pushButton</zorder>
  <zorder>sh_lb</zorder>
  <zorder>back_bt</zorder>
  <zorder>icon_bt</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
